src/Mod/Path/PathScripts/PathSurfaceSupport.py

```diff
@@ -929,22 +929,26 @@ def _calculateOffsetValue(self, isHole, isVoid=False):
         '''_calculateOffsetValue(self.obj, isHole, isVoid) ... internal function.
         Calculate the offset for the Path.Area() function.'''
         self.JOB = PathUtils.findParentJob(self.obj)
-        tolrnc = self.JOB.GeometryTolerance.Value
+        # We need to offset by at least our linear tessellation deflection
+        # (default GeometryTolerance / 4) to avoid false retracts at the
+        # boundaries.
+        tolrnc = max(self.JOB.GeometryTolerance.Value / 10.0,
+                     self.obj.LinearDeflection.Value)
 
         if isVoid is False:
             if isHole is True:
                 offset = -1 * self.obj.InternalFeaturesAdjustment.Value
-                offset += self.radius + (tolrnc / 10.0)
+                offset += self.radius + ()
```

Comment:
We can remove the trailing ` + ()`.  Or, it might need to become ` + tolrnc`.

Reply:
Nice catch! Fixed.

To my surprise, this was not caught by flake8. Got to love the wild world of Python ;)